app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -117,7 +117,7 @@ def search_form_action
 
     def render_visibility_link(document)
       link_to render_visibility_label(document),
-              main_app.edit_curation_concerns_file_set_path(document, anchor: "permissions_display"),
+              main_app.edit_curation_concerns_generic_work_path(document, anchor: "permissions_display"),
```

Comment:
When it's called from here https://github.com/projecthydra/sufia/blob/60326e5de1262176f332fb26b329fe3610097b62/app/helpers/file_set_helper.rb#L19-L25 via https://github.com/projecthydra/sufia/blob/master/app/views/curation_concerns/file_sets/show.html.erb#L19 I'm pretty sure it's a FileSet.